akka-actor/src/main/scala/akka/actor/Actor.scala

```diff
@@ -481,7 +481,8 @@ trait Actor {
    * @param receive current behavior.
    * @param msg current message.
    */
-  protected[akka] def aroundReceive(receive: Actor.Receive, msg: Any): Unit = receive.applyOrElse(msg, unhandled)
+  protected[akka] def aroundReceive(receive: Actor.Receive, msg: Any): Unit =
+    if (receive.isDefinedAt(msg)) receive(msg) else unhandled(msg)
```

Comment:
ah, there is some internal magic there. Then we should use `applyOrElse` but reuse the `default` function